### DISCUSSION TITLE
How is SEO for others using Wiki.js

### DISCUSSION THREAD

**Discussion by jarrodlovas** [requarks/wiki#5312] [Help / Questions]
Good Evening everyone,

Looking to get some help with SEO. 
Unfortunately, I've found the page detection to not be the greatest (I understand this is not WIkijs fault at all but how Google crawls and indexes pages).

What have others done to improve their SEO? At this stage, I'm learning towards hardcoding my sitemap to find my root pages and submitting that sitemap but I'm not 100% sure where to put my sitemap within the web files.

**Comment by ayoahha** [comment]
Hello
Same question here... Seems that SEO is complicated for my wiki. Is there anything we can do to improve this?

**Comment by Philipinho** [comment]
@RodoggA @ayoahha I have just released a sitemap tool for Wiki.js to solve this issue. See  https://github.com/hostwiki/wikijs-sitemap.
